packages/server/src/message/firebase-relay.ts

```diff
@@ -25,11 +25,22 @@ const config = {
   messagingSenderId: '913007764573',
 };
 
-const firebase = firebaseApp.initializeApp(config);
-const messagesRef = firebase.database().ref('messages');
+let firebase;
```

Comment:
What's the purpose of dynamically instantiating this?

Reply:
`initializeApp` should only be called once for a configuration. So I am using the singleton pattern.